From: Max Gurtovoy <maxg@mellanox.com>
Date: Sun, 31 Dec 2017 15:33:27 +0200
Subject: nvme: modify the debug level for setting shutdown timeout
Patch-mainline: v4.16-rc1
Git-commit: 1a3838d732eaae47385490de88d978d4132d3d84
References: FATE#323952, FATE#322506

When an NVMe controller reports RTD3 Entry Latency larger than the value
of shutdown_timeout module parameter, we update the shutdown_timeout
accordingly to honor RTD3 Entry Latency. Use an informational debug level
instead of a warning level for it.

Signed-off-by: Max Gurtovoy <maxg@mellanox.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Acked-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 drivers/nvme/host/core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/nvme/host/core.c
+++ b/drivers/nvme/host/core.c
@@ -2257,7 +2257,7 @@ int nvme_init_identify(struct nvme_ctrl
 						 shutdown_timeout, 60);
 
 		if (ctrl->shutdown_timeout != shutdown_timeout)
-			dev_warn(ctrl->device,
+			dev_info(ctrl->device,
 				 "Shutdown timeout set to %u seconds\n",
 				 ctrl->shutdown_timeout);
 	} else
